generators/swift/codegen/src/ast/Enum.ts

```diff
@@ -0,0 +1,59 @@
+import { AstNode, Writer } from "@fern-api/generator-commons";
+import Swift, { AccessLevel, EnumCase, Type } from "..";
+
+export declare namespace Enum {
+    interface Args {
+        accessLevel?: AccessLevel;
+        name: string;
+        inheritance?: Type[],
+        enumCases: EnumCase[]
```

Comment:
very small nit + follow up: the ir calls this `enumValues` so i think easier to align